modules/ROOT/pages/fd-scatter-gather.adoc

```diff
@@ -0,0 +1,259 @@
+<4> A Transform card for mapping the fields in the variable to the fields in a CSV file.
+
+First, you create the custom data type like this:
+
+.The *Edit Custom Type* dialog lets you create the custom data type.
+image::fd-sg-6-custom-type.png[The dialog for creating the custom data type]
+<1> The *Format* field is set to *CSV*.
+<2> The *Type* field is set to *From schema*.
+<3> The schema is this:
+----
+#%RAML 1.0 DataType
+type: array
+items:
+  properties:
+    Name:
+      type: string
+    ID:
+      type: string
+    Email:
+      type: string
+----
+
+Next, you add the HTTP Listener card, which listens at the path that you specify in the *Path* field. The name of the path is given to the name of the CSV file that is downloaded to your computer:
+
+.The *Path* field is in the *General* settings.
+image::fd-sg-2-http-listener.png[Location of the Path field]
+
+Though this example uses an HTTP Listener card, you can send data to a Scatter-Gather card from any card.
+
+Next, you add the Scatter-Gather card and place an instance of Salesforce Connector in each of two routes. The instances of Salesforce Connector use different configurations because they connect to different Salesforce organizations, and therefore require different connection information.
+
+You specify this query in the *Salesforce Query* field in each instance:
+
+.The *Salesforce Query* field is in the *General* settings.
+image::fd-sg-3-sf-query.png[Query and location of the Salesforce Query field]
+
+The message that leaves the Scatter-Gather card has these parts:
+
+----
+payload
+  0
+    payload
+      Name
+      ID
+      Email
+  1
+    payload
+      Name
+      ID
+      Email
+attributes
+vars
+----
+
+The payload from the Scatter-Gather card consists of two sub-payloads, one payload for each route in the card. The sub-payload under `0` is the payload from the first route in the card, and the sub-payload under `1` is the payload from the second route.
+
+It is possible at this point in the flow to combine the two sub-payloads into a CSV file with a DataWeave script. However, if you would rather not write a DataWeave script, you can concatenate the two sub-payloads into a single payload that you store in a variable, and then map the fields in the single payload to the fields in a CSV file.
+
+The Set Variable card includes 1) a name for the variable and 2) an expression in the *Value* field. The expression (at 3) is this:
+
+----
+payload["0"]["payload"] ++ (payload["1"]["payload"])
+----
+
+You can create this expression by dragging the *Custom Expression* pill from the right side of the card into the *Value* field, clicking the arrow in the pill, and clicking *Expand*. Then, drag the first *Payload* pill into the text box, add the two plus signs, then drag the second *Payload* pill into the text field.
+
+.The *Set Variable* card looks like this.
+image::fd-sg-4-set-variable.png[]
+<1> The name specified for the variable.
```

Comment:
Because none of these is a complete sentence, we don't need periods. Unless you are going for absolute consistency.